  File "/home/mohamed/.local/share/uv/tools/pyinfra/lib/python3.10/site-packages/pyinfra_cli/cli.py", line 243, in cli
    _main(*args, **kwargs)
  File "/home/mohamed/.local/share/uv/tools/pyinfra/lib/python3.10/site-packages/pyinfra_cli/cli.py", line 353, in _main
    inventory = make_inventory(
  File "/home/mohamed/.local/share/uv/tools/pyinfra/lib/python3.10/site-packages/pyinfra_cli/inventory.py", line 173, in make_inventory
    return make_inventory_from_files(inventory, override_data, cwd, group_data_directories)
  File "/home/mohamed/.local/share/uv/tools/pyinfra/lib/python3.10/site-packages/pyinfra_cli/inventory.py", line 260, in make_inventory_from_files
    groups = _get_groups_from_filename(inventory_filename)
  File "/home/mohamed/.local/share/uv/tools/pyinfra/lib/python3.10/site-packages/pyinfra_cli/inventory.py", line 91, in _get_groups_from_filename
    attrs = exec_file(inventory_filename, return_locals=True)
  File "/home/mohamed/.local/share/uv/tools/pyinfra/lib/python3.10/site-packages/pyinfra_cli/util.py", line 57, in exec_file
    code = compile(code_str, filename, "exec")
  File "inventories/docker.py", line 1
    inventory = (
IndentationError: unexpected indent
